codeforamerica/vita-min
# nj-189 taxable income bug

## Link to pivotal/JIRA issue

https://github.com/newjersey/affordability-pm/issues/189

## Is PM acceptance required?
- No - merge after code review approval

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- The test case was failing because `interest_on_gov_bonds` was `0` which made line 16a `nil`. We want to allow `0`, and since negative isn't possible as it's a sum of amounts paid, we don't need a check there.
- Adjusted test expectation and removed an extra test no longer needed.
## How to test?
- Use `test_togo_dores1` and you should now see `520` for `TaxableInterestIncome` on line 16a (was empty before this fix).
## Screenshots (for visual changes)
<img width="485" alt="image" src="https://github.com/user-attachments/assets/5caa0159-2ab9-4ec3-9648-49d6258867f3">
<img width="666" alt="image" src="https://github.com/user-attachments/assets/6d6265a2-0164-4cb1-a7be-e1f23a1df451">
